PosteAPI/migrations/0002_remove_folder_shared_users_folderpermission.py

```diff
@@ -1,28 +0,0 @@
-# Generated by Django 4.2.5 on 2023-10-15 02:14
```

Comment:
I would recommend updating the README that we are going to delete migrations for the sake of simplicity.